src/pages/ocpDashboard/ocpDashboard.tsx

```diff
@@ -35,7 +33,7 @@ const OcpDashboardBase: React.SFC<OcpDashboardProps> = ({
   providersFetchStatus,
   widgets,
 }) => (
-  <div className={theme}>
```

Comment:
You've removed the theme styles for the OCP dashboard, but it's still used for awsDashboard.tsx? They should be the same, since OCP is a copy of the AWS dashboard.

Reply:
Oops, removed that on accident.